### PR TITLE
Completely hide widget if there is nothing to show

### PR DESCRIPTION
**Pull request by luisbocanegra**
refs: #58

### CODE REVIEW

File: package/contents/ui/main.qml [hunk 1]
```diff
@@ -28,11 +28,16 @@ PlasmoidItem {
...
     property bool vertical: plasmoid.formFactor === PlasmaCore.Types.Vertical
     property bool leftEdgeLocation: plasmoid.location === PlasmaCore.Types.LeftEdge
+    property bool hideWidget: !tasksModel.hasActiveWindow && plasmoid.configuration.widgetElementsDisabledMode === WidgetElement.DisabledMode.Hide
+    property bool inEditMode: Plasmoid.containment.corona?.editMode ?? false
 
     signal invokeKWinShortcut(string shortcut)
     signal widgetElementsLayoutUpdated
 
     Plasmoid.constraintHints: Plasmoid.CanFillArea
+    Plasmoid.status: hideWidget
```
Comment (antroids) [inline package/contents/ui/main.qml:38]: It seems the `status` property is just tells to container that the plasmoid is hidden.
Let's remove the `emptyWidgetPlaceholder` to hide it.
This line does not require changes.

**Review by antroids — CHANGES_REQUESTED** [review]
Thank you for the PR and all your contributions!

File: package/contents/ui/main.qml [hunk 2]
```diff
@@ -28,11 +28,16 @@ PlasmoidItem {
     property bool widgetHovered: widgetHoverHandler.hovered
     property bool vertical: plasmoid.formFactor === PlasmaCore.Types.Vertical
     property bool leftEdgeLocation: plasmoid.location === PlasmaCore.Types.LeftEdge
+    property bool hideWidget: !tasksModel.hasActiveWindow && plasmoid.configuration.widgetElementsDisabledMode === WidgetElement.DisabledMode.Hide
+    property bool inEditMode: Plasmoid.containment.corona?.editMode ?? false
```
[the remark this answers is not in the document]
Comment (luisbocanegra) [reply package/contents/ui/main.qml]: done

File: package/contents/ui/main.qml (hunk 1 above)
Comment (luisbocanegra) [reply package/contents/ui/main.qml:38]: Done

**Comment by antroids** [comment]
Merged, thanks again.

**Pull request merged**